### PR TITLE
Removed no sources in simulation log info.

### PR DESCRIPTION
**Pull request by e-g-melo**
The "No sources in simulation" log info was commented out to avoid showing this message in GUI.

image

### CODE REVIEW

**Comment by e-g-melo** [comment]
Actually, I will resubmit this following these instructions (https://docs.flexcompute.com/projects/tidy3d/en/latest/development/index.html).

**Comment by e-g-melo** [comment]
Hi! I am getting this error after commenting out that info log:
image

Should I remove this?
image

File: tests/data/invdes_history.json
Comment (momchil-flex) [inline tests/data/invdes_history.json:1]: @tylerflex should we add this to gitignore?

**Review by momchil-flex — APPROVED** [review]
I think this is good, we can just squash and merge but I think we should add the `invdes_history.json` file to gitignore and remove from this PR. Tyler and I could do it once he confirms.

File: tests/data/invdes_history.json
Comment (tylerflex) [reply tests/data/invdes_history.json:1]: sounds good to me

**Review by tylerflex — APPROVED** [review]
Thanks @e-g-melo The PR looks ok to me. Could you file an issue (if not already) so we can make sure to fix this in the long term? Not sure who to assign at this point.

**Pull request merged**